.github/workflows/test-ivy-cron.yml

```diff
@@ -2,7 +2,7 @@ name: test-ivy-cron
 on:
   workflow_dispatch:
   schedule:
-    - cron: '30 * * * *'
+    - cron: '0 8 * * *'
```

Comment:
Question: Was this cron change on purpose? I think if we are going to change it we might need to talk with the CI team, otherwise we can just revert this changes.